README.md

```diff
@@ -123,3 +123,38 @@ $ heroku config:set LINE_CHANNEL_TOKEN=<Your channel token>
 $ heroku config:set BOTIMIZE_API_KEY=<Your botimize API Key (optional)>
 $ heroku config:set GOOGLE_CREDENTIALS=<Your google credential (optional)>
 ```
+
+## Google Analytics Events table
+
+Sent event format: `Event category` / `Event action` / `Event label`
+
+1. User submits article
+  - `UserInput` / `MessageType` / `<text | image | video | ...>`
+
+  - If articles found:
+    - `UserInput` / `ArticleSearch` / `ArticleFound`
+    - `Article` / `Search` / `<article id>` for each article found
+  - If article too short:
```

Comment:
How about **If message too short (to found articles)** ?

Reply:
Maybe I'll go with "inputted message does not look like messages being forwarded in instant messenger"? That's more precise.